apps/client/src/app/app.component.html

```diff
@@ -150,6 +150,9 @@
           <li>
             <a href="../nl" title="Ghostfolio in Nederlands">Nederlands</a>
```

Comment:
Even though this wasn't changed in this PR,  given all the other options mention the language itself (not the country) should this be `Ghostfolio in Dutch`?

Reply:
I think it's fine like this 😃  See [here](https://translate.google.com/?sl=en&tl=nl&text=Dutch&op=translate)!

Reply:
Ah. Did not know this. Thanks for sharing 🙌